### PR TITLE
Add exostar certificates

### LINKED ISSUE #322: Exostar PKI not supported
Hi,
I am trying to attach my Exostar PKI certificate to my login.gov account but I get the error that it is not valid.
Is this intended?
Thanks,
Mark

### PR DESCRIPTION
**Pull request by mitchellhenke**
Closes #322 

The Exostar certificates are copied from the FICAM bundle

### CODE REVIEW

File: config/application.yml.default [hunk 1]
```diff
@@ -9,7 +9,7 @@ aws_http_timeout: '5'
 http_read_timeout: '5'
 http_open_timeout: '5'
 
-ca_issuer_host_allow_list: 'aia.certipath.com,repo.fpki.gov,crl.disa.mil,http.fpki.gov,ssp-aia.symauth.com,pki.treasury.gov,rootweb.managed.entrust.com,aia1.ssp-strong-id.net,pki.treas.gov,crls.pki.state.gov,sspweb.managed.entrust.com,nfirootweb.managed.entrust.com,ssp-aia.digicert.com'
+ca_issuer_host_allow_list: 'aia.certipath.com,repo.fpki.gov,crl.disa.mil,http.fpki.gov,ssp-aia.symauth.com,pki.treasury.gov,rootweb.managed.entrust.com,aia1.ssp-strong-id.net,pki.treas.gov,crls.pki.state.gov,sspweb.managed.entrust.com,nfirootweb.managed.entrust.com,ssp-aia.digicert.com,www.fis.evincible.com'
```
Comment (zachmargolis) [inline config/application.yml.default:12]: do we need to update the domain allowlist for this?

**Review by zachmargolis — APPROVED** [review]
LGTM

File: config/application.yml.default (hunk 1 above)
Comment (mitchellhenke) [reply config/application.yml.default:12]: The production/deployed environment list is below, and I think we override it in `production` via S3 too so it's not strictly necessary.

Comment (zachmargolis) [reply config/application.yml.default:12]: ok sweet, I always forget which configs in this repo get blocked by the firewall and which don't

Comment (markrosenbaum) [reply config/application.yml.default:12]: Hi, I may be wrong but on an Exostar certificate the domain is marked as fis.evincible.com not www.fis.evincible.com, they may be interchangeable but I just wanted to make sure.

Comment (mitchellhenke) [reply config/application.yml.default:12]: This is the list for development and stuff where there isn't a firewall and has tests run against it to make sure we aren't missing any hosts by default (and as a reference to make updates to the deployed environment allowlist if needed)

**Pull request merged**